### DISCUSSION TITLE
moveTab commands explanations

### DISCUSSION THREAD

**Discussion by tamwile** [gdh1995/vimium-c#1117] [Q&A]
Hello, thank you for this webextension.
I have a couple of questions regarding the moveTab family of commands.

Is it possible to use the moveTab commands ( moveTabLeft and moveTabRight ) with filter="host" ? 
Can we move multiple tabs at once ? (like if i select multiple tabs with ctrl + click, and then drag them manually )
What is the "group" parameter ? Is this referring to containers ?
Ideally, i would like to take all tabs from one domain and move them to the beginning (before the first tab) or at the end (after the last tab).
Is this possible ?

**Comment by gdh1995** [comment]
No `moveTab Left/Right` don't support `filter`. `group` is used by `moveTab Left/Right` to decide whether to move a tab from a group to another - it won't move a tab across groups unless `group=false`.

You may try `moveTabToNewWindow`:
```
run <f9> moveTabToNewWindow#all&filter=host+-9999moveTabToNextWindow#last&tabs&limited=false&position=begin
```

`<f9>` may be replaced with other keys
